cardano-db/src/Cardano/Db/Multiplex.hs

```diff
@@ -105,10 +105,11 @@ runExtraMigrations trce blockNoDiff PruneConsumeMigration {..} = do
           ExtraCons.migrateTxOut (Just trce)
         (False, _, True) -> do
           shouldInsertToMigrationTable
+          liftIO $ logInfo trce "Now Running extra migration prune tx_out"
+          ExtraCons.deleteAndUpdateConsumedTxOut trce blockNoDiff
+
           liftIO $ logInfo trce "Running extra migrations consumed_tx_out and prune tx_out"
           ExtraCons.migrateTxOut (Just trce)
```

Comment:
Since `deleteAndUpdateConsumedTxOut` also does the migration, we may not need calling `migrateTxOut`